main.js

```diff
@@ -74,5 +79,9 @@ function placeHours(hours, minHours, maxHours, lineStartX, lineEndX){
 $(document).ready(function(){
   drawNumberline();
 
+  $('input.hours_input').on('change', function(){
```

Comment:
That is, we don't just care that they're two digit numbers, we care that they're multiples of 0.25 hours.